roles/reproducer/tasks/configure_controller.yml

```diff
@@ -300,3 +292,67 @@
       ansible.builtin.copy:
         dest: "/home/zuul/reproducer-variables.yml"
         content: "{{ _filtered_vars | to_nice_yaml }}"
+
+    - name: Get interfaces-info content
+      register: _nic_info
+      ansible.builtin.slurp:
+        src: "{{ _ctl_reproducer_basedir }}/parameters/interfaces-info.yml"
+
+    # We detected OCP cluster may have some downtime even after it's supposed
+    # to be started.
+    # It means everything was right from the libvirt_manager point of view, it
+    # could get access, configure the needed bits and move on.
+    # But when we hit this step, cluster may be unresponsive - suspecting
+    # some NetworkManager or related tool doing some "late" tasks.
+    # This task here is to ensure all of the OCPs nodes are ready to be
+    # consumed by the networking_mapper - it will connect to all of the nodes
+    # in order to gather some facts of interest.
+    - name: Wait for OCP nodes to be ready
+      delegate_to: "{{ item }}"
+      ansible.builtin.wait_for_connection:
+        sleep: 2
+        timeout: 300
+      loop: "{{ groups.ocps }}"
+
+    - name: Generate networking definition
+      vars:
+        cifmw_networking_mapper_ifaces_info: >-
+          {{ _nic_info.content | b64decode | from_yaml }}
+        cifmw_networking_mapper_network_name: >-
+          {{ _layout.vms.controller.nets.1 }}
+      ansible.builtin.import_role:
+        name: networking_mapper
+
+    - name: Generate configMap snippet
+      vars:
+        cifmw_networking_mapper_config_generator_output: >-
+          /home/zuul/ci-framework-data/artifacts/ci_k8s_snippets
+      ansible.builtin.import_role:
+        name: networking_mapper
+        tasks_from: network_config_values.yml
+
+    - name: Generate values.yaml for EDPM deploy
```

Comment:
follow-up:
- change title to something like "Generate controlplan data"
- set the target file to something like "controlplan-values.yaml"

Reply:
done